doc/plugins.dox

```diff
@@ -186,59 +195,143 @@ Let's start with the very basic headers and add more later if necessary.
 @a geanyplugin.h includes all of the Geany API and also the necessary GTK header files,
 so there is no need to include @a gtk/gtk.h yourself.
 
-@note
-@a plugindata.h contains the biggest part of the plugin API and provides some basic macros.
-
-Then you should define two basic variables which will give access to data fields
-provided by the plugin API.
-@code
-GeanyPlugin			*geany_plugin;
-GeanyData			*geany_data;
-@endcode
+@note @a plugindata.h contains the biggest part of the plugin API and provides some basic macros.
```

Comment:
Does this have to be imported by the user? If not why mention it, if so then it needs to be specified.